docs/src/main/sphinx/connector/hive.rst

```diff
@@ -174,13 +174,13 @@ Configuration
 -------------
 
 Create ``etc/catalog/hive.properties`` with the following contents
-to mount the ``hive-hadoop2`` connector as the ``hive`` catalog,
+to mount the ``hive`` connector as the ``hive`` catalog,
```

Comment:
Are these really the only instances of the connector name in all the docs?

Reply:
Yes